src/Octoshift/Services/OctoLogger.cs

```diff
@@ -33,8 +33,9 @@ public class OctoLogger
     public OctoLogger()
     {
         var logStartTime = DateTime.Now;
-        _logFilePath = $"{logStartTime:yyyyMMddHHmmss}.octoshift.log";
-        _verboseFilePath = $"{logStartTime:yyyyMMddHHmmss}.octoshift.verbose.log";
+        var processId = Environment.ProcessId;
```

Comment:
I did a quick bit of a research on this to make sure that we weren't going to get a value that would lead to an invalid filename. `Environnment.ProcessId` is an Int32 according to the [.NET docs](https://learn.microsoft.com/en-us/dotnet/api/system.environment.processid?view=net-7.0#system-environment-processid), so this should be fine.